library.gradle

```diff
@@ -6,6 +6,27 @@ android {
     compileSdkVersion compileVersion
     buildToolsVersion "30.0.3"
 
+    flavorDimensions "extraAPIs"
+    productFlavors {
+        defaults {
+            dimension "extraAPIs"
+        }
+        integrationTest {
+            dimension "extraAPIs"
+            minSdkVersion 21 // Needed because mockk-android requires API 21
+            compileSdkVersion 31
```

Comment:
Had to increase this version as well due to incompatibilities with some of the testing libraries we use...